src/App.module.scss

```diff
@@ -1,5 +0,0 @@
-.leftPanelOverride {
```

Comment:
I wonder if you were seeing the slowness because I was demoing to myself how style overriding would work technically. For now it can just be normal css like this to override, but if you think there should be a reusable transition CSS vaer that all things share, maybe overriding transition can also be a global theme config. (which we can still override in a class like this deleted one)